MIGRATING.md

```diff
@@ -2,6 +2,94 @@
 
 This library uses semantic versioning and follows Okta's [library version policy](https://developer.okta.com/code/library-versions/). In short, we don't make breaking changes unless the major version changes!
 
+## Migrating to 3.0
+
+Version 3.0 of this library introduces a number of breaking changes from previous versions; in addition to new classes some class definitions are no longer backward compatible due to method renames and signature changes, see [Breaking Changes](#breaking-changes).
+
+### Breaking Changes
+
+The following is a list of changes that break backward compatibility in version 3.0.
+
+**Okta.Sdk.OktaClient**
+- `CreatedScoped(Okta.Sdk.RequestContext requestContext)`
+<br />&mdash; Renamed `CreateScoped(Okta.Sdk.RequestContext requestContext)`
+
+**Okta.Sdk.GroupsClient**                
+- `ListGroups(String q,String filter,String after,Int32 limit,String expand)` 
+<br />&mdash; Signature changed `ListGroups(String q,String filter,String after,Int32 limit)`
```

Comment:
Question: Are we using `Int32` or `int`?

Reply:
Same here, reflection; I'll update.